### ISSUE TITLE
Cloud Build fails due to visualization server build configuration error

### ISSUE THREAD

**Issue by neuromage** [kubeflow/pipelines#1839]
Cloud build is failing due to this error:
```
ERROR
ERROR: failed to find one or more images after execution of build steps: ["gcr.op/ml-pipeline-test/visualization-server:68138ec891eacffb1d7edb2609cde5c22166cf00"]
```

The line here seems to be the issue:
https://github.com/kubeflow/pipelines/commit/48173ad7760d61a769af56f735ce959d1f13ffe6#r34693590

**Comment by ajchili** [comment]
Should be fixed by #1816. Currently waiting to be merged.

**Comment by neuromage** [comment]
Duplicate of #1815
